cns/configuration/configuration.go

```diff
@@ -46,7 +46,7 @@ func ReadConfig() (CNSConfig, error) {
 	// Check if env set for config path otherwise use default path
 	configpath, found := os.LookupEnv("CNS_CONFIGURATION_PATH")
 	if !found {
-		dir, err := filepath.Abs(filepath.Dir(os.Args[0]))
+		dir, err := os.Getwd()
```

Comment:
It was stated that "Getwd returns a rooted path name corresponding to the current directory". So if we run the exe from different directory, will this work?